test/kitchen/tasks/apt-test-6.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -l
```

Comment:
can you add a short explanation on each of these scripts to explain their purpose?

Reply:
Yes I can